docs-src/cluster-ops-context/manage-custom-search-attributes.md

```diff
@@ -0,0 +1,18 @@
+---
+id: manage-custom-search-attributes
+title: Managing custom Search Attributes
+description: You can create custom Search Attributes. On a self-hosted Temporal Cluster, you can remove them; on Temporal Cloud, you can rename them.
+
+sidebar_label: Custom Search Attributes
+tags:
+  - guide-context
+---
+
+To manage your custom Search Attributes on Temporal Cloud, use `tcld`. With Temporal Cloud, you can create and rename custom Search Attributes.
+
+To manage your custom Search Attributes on self-hosted Temporal Clusters, use `tctl`. With self-hosted Temporal Cluster, you can create and remove custom Search Attributes.
+Note that if you use [SQL databases](/clusters/how-to-set-up-visibility-in-a-temporal-cluster) with Temporal Server v1.20 and later, creating a custom Search Attribute creates a mapping with a database field name in the Visibility store `custom_search_attributes` table.
+Removing a custom Search Attribute removes this mapping with the database field name but does not remove the data.
```

Comment:
Should we add this warning to "how to remove Custom Search Attributes" as well ?

Reply:
was added in a sneak update just before this.